Update txt2tags.py

goetzschnell opened the pull request:
added target ConTeXt (ctx),
topics to discuss eventually:
- escaping
- some rules
- output (layout and design of ConTeXt output)

jendrikseipp commented on txt2tags.py:
```diff
@@ -82,7 +82,7 @@
 my_name = "txt2tags"
 my_email = "jendrikseipp@gmail.com"
-__version__ = "3.7"
+__version__ = "3.7.1"
```
No need to change the version.

goetzschnell replied:
eliminated

jendrikseipp commented on txt2tags.py:
```diff
@@ -404,7 +405,59 @@
-"""
+""",
+   "ctx": \
+r"""%% created from txt2tags
```
No need for the comment.

goetzschnell replied:
eliminated

goetzschnell commented:
eliminated version and comment,
added samples/sample.ctx and samples/sample.pdf
ran run.py, added run.pyc in every test folder
result of run.py:
[2020-02-12result.txt](https://github.com/txt2tags/txt2tags/files/4194790/2020-02-12result.txt)

goetzschnell commented:
eliminated sample.pdf
eliminated *.pyc

jendrikseipp commented:
Merged. Thanks!
